### ISSUE TITLE
fredr() and parameter vintage_dates

### ISSUE THREAD

**Issue by clgarciga** [sboysel/fredr#88]
From https://cran.r-project.org/web/packages/fredr/fredr.pdf under `fred_series_observations`:

![image](https://user-images.githubusercontent.com/13739205/103041097-671cd400-4543-11eb-8a15-54201fa2fcd5.png)

However, it appears that `fredr` cannot accept more than one date for parameter `vintage_dates`:

![image](https://user-images.githubusercontent.com/13739205/103041138-86b3fc80-4543-11eb-9c77-fb68fc3f0f7a.png)

Ideally, when passing multiple dates to `vintage_dates`, a fourth column in the return tibble called, for example, `vintage_date` would denote which vintage that data point is from.

**Comment by sboysel** [comment]
Nice catch!  The API parameter itself accepts a comma separated string of dates but it seems we didn't accommodate for that in the `vintage_dates` parameter of `fredr()`.  I can take a stab at this shortly.

**Comment by clgarciga** [comment]
Thanks! And, thanks for the quick response!

**Comment by sboysel** [comment]
Should be fixed with #89

``` r
fredr::fredr(series_id = "GDPC1", vintage_dates = as.Date(c("1991-12-04", "1991-12-20")))
#> # A tibble: 132 x 3
#>    date       series_id value
#>    <date>     <chr>     <dbl>
#>  1 1959-01-01 GDPC1     1910.
#>  2 1959-04-01 GDPC1     1941.
#>  3 1959-07-01 GDPC1     1932.
#>  4 1959-10-01 GDPC1     1943.
#>  5 1960-01-01 GDPC1     1978.
#>  6 1960-04-01 GDPC1     1974.
#>  7 1960-07-01 GDPC1     1976.
#>  8 1960-10-01 GDPC1     1965.
#>  9 1961-01-01 GDPC1     1980.
#> 10 1961-04-01 GDPC1     2009.
#> # … with 122 more rows
```

<sup>Created on 2021-01-04 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by sboysel** [comment]
This change has now been merged the master branch with #89.  Will do another CRAN release soon but in the meantime, you can install with `devtools::install_github()`.

**Comment by clgarciga** [comment]
Thank you!
